docs/security/access-control.md

```diff
@@ -0,0 +1,659 @@
+| CreateRole | Metalake                  | Create a role |
+
+### Permission privileges
+
+| Name         | Supports Securable Object | Operation              |
+|--------------|---------------------------|------------------------|
+| ManageGrants | Metalake                  | grant or revoke a role |
+
+### Catalog privileges
+
+| Name          | Supports Securable Object | Operation        |
+|---------------|---------------------------|------------------|
+| CreateCatalog | Metalake                  | Create a catalog |
+| UseCatalog    | Metalake, Catalog         |                  |
+
+:::info
+
+`USE_CATALOG` is needed for a user to interact with any object within the catalog. 
+
+For example, to select data from a table, users need to have the SELECT_TABLE privilege on that table and
+`USE CATALOG` privileges on its parent catalog as well as `USE SCHEMA` privileges on its parent schema.
+
+:::
+
+
+### Schema privileges
+
+| Name         | Supports Securable Object | Operation       |
+|--------------|---------------------------|-----------------|
+| CreateSchema | Metalake, Catalog         | Create a schema |
+| UseSchema    | Metalake, Catalog, Schema | Use a schema    |
+
+:::info
+
+`UseSchema`is needed for a user to interact with any object within the schema. 
+
+For example, to select data from a table, users need to have the `SELECT_TABLE` privilege on that table
+and `USE SCHEMA` privileges on its parent schema.
+
+:::
+
+### Table privileges
+
+| Name        | Supports Securable Object         | Operation                                      |
+|-------------|-----------------------------------|------------------------------------------------|
+| CreateTable | Metalake, Catalog, Schema         | Create a table                                 |
+| ModifyTable | Metalake, Catalog, Schema, Table  | Use the SQL `UPDATE`,`DELETE`,`INSERT` a table |
+| SelectTable | Metalake, Catalog, Schema, Table  | Use the SQL `SELECT` data from a table         |
+
+### Topic privileges
+
+| Name         | Supports Securable Object        | Operation                                 |
+|--------------|----------------------------------|-------------------------------------------|
+| CreateTopic  | Metalake, Catalog, Schema        | Create a topic                            |
+| ProduceTopic | Metalake, Catalog, Schema, Topic | Produce a topic (including alter a topic) |
+| ConsumeTopic | Metalake, Catalog, Schema, Topic | Consume a topic                           |
+
+### Fileset privileges
+
+| Name          | Supports Securable Object          | Operation                                   |
+|---------------|------------------------------------|---------------------------------------------|
+| CreateFileset | Metalake, Catalog, Schema          | Create a fileset                            |
+| WriteFileset  | Metalake, Catalog, Schema, Fileset | Write a fileset (including alter a fileset) |
```

Comment:
What's the difference between `CreateFileset` and `WriteFileset`?

Reply:
You create any fileset. But if others create another fileset. You don't have write privilege, too.